packages/create-mc-app/src/hint-outdated-version.js

```diff
@@ -1,17 +1,27 @@
 const semver = require('semver');
-const execSync = require('child_process').execSync;
-const pkg = require('../../package.json');
-const logger = require('../logger');
+const execa = require('execa');
```

Comment:
We now use this library, which wraps the child_process and provides a nicer interface to work with, also with promises

Reply:
I presume we could use it most cases instead of shelljs. In case we still use it?

Reply:
Hmm maybe yeah, although I like shelljs as well 🙂